src/org/anjocaido/groupmanager/utils/BukkitWrapper.java

```diff
@@ -157,7 +157,7 @@ public OfflinePlayer getOfflinePlayer(UUID uid) {
 	 * Attempts to match any players with the given name, and returns a list of all possible matches.
 	 * This list is not sorted in any particular order. If an exact match is found, the returned list will only contain a single result.
 	 * 
-	 * @param the (partial) name to match
+	 * @param name to match

```

Comment:
This is a partial name match.

Reply:
IntelliJ refused to stop complaining otherwise, but I'll add the `(partial)` after the `name`